### PR TITLE
Clean up unused manifests

### PR DESCRIPTION
**Pull request by hongkailiu**
`auth_proxy*.yaml` and `leader-election-role*.yaml` should be gone with `config/default/manager_auth_proxy_patch.yaml` deleted by https://github.com/openshift/cincinnati-operator/pull/65/commits/ce9a44775fe7473d3d4c60b88154ec73d95a6562#diff-7263fc341f1b59afdc8599e1f570cdd180c28bf8cca71ecee1e58b58131202c5

### CODE REVIEW

**Comment by hongkailiu** [comment]
https://issues.redhat.com/browse/OTA-1166

**Comment by hongkailiu** [comment]
I was naive. Those manifests are generated by operator-sdk while the project was initialized. 

/close

File: config/rbac/role_binding.yaml [hunk 1]
```diff
@@ -1,14 +1,3 @@
...
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: updateservice-operator
-subjects:
-- kind: ServiceAccount
-  name: updateservice-operator
-roleRef:
-  kind: Role
-  name: updateservice-operator
-  apiGroup: rbac.authorization.k8s.io
```
Comment (hongkailiu) [inline config/rbac/role_binding.yaml:11]: The `roleRef` has been removed with https://github.com/openshift/cincinnati-operator/commit/3990db30086455967c2ce5e7deba4247c4fa8b14#diff-6d4c4e29ec096149781536bb9118fde8242c1901df764e3eda3019ac8503d6c7

Comment (wking) [reply config/rbac/role_binding.yaml:11]: Maybe we want to keep the Roles and such and move away from the ClusterRoleBinding and such now that we no longer support `All namespaces on the cluster` installs (1137a5c057902, #134, moving `AllNamespaces` to `supported: false`)?

image

**Review by wking — APPROVED** [review]
/lgtm

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**